<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_19">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_18">
      <item>
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 127);
background-color: rgb(199, 187, 183);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>CSV Register Parser Tool</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="font">
             <font>
              <family>Verdana</family>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QGroupBox {
    background-color: #f0f0f0;
    border: 2px solid #cccccc;
    border-radius: 5px;
    margin-top: 20px; /* Adjust to move the title up or down */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* Position at the top center */
    padding: 0 10px;
    color: #333;
    font: bold 14px &quot;Arial&quot;;
    background-color: transparent; /* Ensure the title background is transparent */
}</string>
            </property>
            <property name="title">
             <string>Register Definition</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Filename</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Address Header</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Width Header</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_11">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Encoding</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <widget class="QLineEdit" name="definition_filename_lineEdit">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                      </font>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="definition_header_lineEdit">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                      </font>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="definition_width_lineEdit">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                      </font>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="definition_encoding_lineEdit">
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="browse_definition_file_button">
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Browse..</string>
                 </property>
                 <property name="icon">
                  <iconset resource="GUI_Logos/images.qrc">
                   <normaloff>:/logos/read_register_icon.png</normaloff>:/logos/read_register_icon.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="font">
             <font>
              <family>Verdana</family>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QGroupBox {
    background-color: #f0f0f0;
    border: 2px solid #cccccc;
    border-radius: 5px;
    margin-top: 20px; /* Adjust to move the title up or down */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* Position at the top center */
    padding: 0 10px;
    color: #333;
    font: bold 14px &quot;Arial&quot;;
    background-color: transparent; /* Ensure the title background is transparent */
}</string>
            </property>
            <property name="title">
             <string>Register Values</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_8">
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Filename</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_8">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Offset Header</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_9">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Values Header</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_15">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Encoding</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_9">
                   <item>
                    <widget class="QLineEdit" name="values_filename_lineEdit">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                      </font>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="values_offset_lineEdit">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                      </font>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="values_header_lineEdit">
                     <property name="font">
                      <font>
                       <family>Verdana</family>
                      </font>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="values_encoding_lineEdit">
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="browse_values_file_button">
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                  </font>
                 </property>
                 <property name="text">
                  <string>Browse..</string>
                 </property>
                 <property name="icon">
                  <iconset resource="GUI_Logos/images.qrc">
                   <normaloff>:/logos/read_register_icon.png</normaloff>:/logos/read_register_icon.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
    background-color: #f0f0f0;
    border: 2px solid #cccccc;
    border-radius: 5px;
    margin-top: 20px; /* Adjust to move the title up or down */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* Position at the top center */
    padding: 0 10px;
    color: #333;
    font: bold 14px &quot;Arial&quot;;
    background-color: transparent; /* Ensure the title background is transparent */
}</string>
          </property>
          <property name="title">
           <string>Output File</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>HTML Filename:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_12">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>CSV Filename</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_15">
                 <item>
                  <widget class="QLineEdit" name="html_output_file_lineEdit">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                    </font>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="csv_file_output_lineEdit">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                    </font>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QPushButton" name="parse_input_files_button">
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                  </font>
                 </property>
                 <property name="text">
                  <string>Parse</string>
                 </property>
                 <property name="icon">
                  <iconset resource="GUI_Logos/images.qrc">
                   <normaloff>:/logos/write_icon.png</normaloff>:/logos/write_icon.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="analyze_parsed_data_button">
                 <property name="text">
                  <string>Analyze Data</string>
                 </property>
                 <property name="icon">
                  <iconset resource="GUI_Logos/images.qrc">
                   <normaloff>:/logos/eye_scan.png</normaloff>:/logos/eye_scan.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUser_Manual">
    <property name="title">
     <string>User Manual</string>
    </property>
    <addaction name="openManual_QAction"/>
   </widget>
   <addaction name="menuUser_Manual"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openManual_QAction">
   <property name="text">
    <string>Open Manual</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="GUI_Logos/images.qrc"/>
 </resources>
 <connections/>
</ui>
